include/aspect/material_model/interface.h

```diff
@@ -217,6 +217,16 @@ namespace aspect
                           const bool use_strain_rates = true);
 
       /**
+       * Copy Constructor.
+       */
+      MaterialModelInputs(const MaterialModelInputs<dim> &material);
+
+      /**
+       * Assignment operator.
+       */
+      MaterialModelInputs &operator=(const MaterialModelInputs &material);
```

Comment:
The error message is:
`An error occurred in line <115> of file </home/dannberg/software/dealii-8.5-install/include/deal.II/dofs/dof_accessor.templates.h> in function
    void dealii::DoFAccessor<structdim, DoFHandlerType, lda>::set_dof_handler(DoFHandlerType*) [with int structdim = 2; DoFHandlerType = dealii::DoFHandler<2>; bool level_dof_access = false]
The violated condition was: 
    dh != 0
Additional information: 
    (none)`

Basically, I think it tries to set a dof_handler, but does not have one.